Update testing steps.

wallclockbuilder opened the pull request:
Instructions for setting up a self-contained local Bisq network on Bitcoin regtest have been updated. 
`dev-setup.md` is deprecated. `Makefile` is new new version. 

Reference: https://github.com/bisq-network/bisq/blob/master/docs/README.md

wallclockbuilder committed:
Update testing steps.


Instructions for setting up a self-contained local Bisq network on Bitcoin regtest have been updated. 
`dev-setup.md` is deprecated. `Makefile` is new new version. 

Reference: https://github.com/bisq-network/bisq/blob/master/docs/README.md

ripcurlx reviewed (changes requested):
NACK - path is wrong. There is not Makefile.md and I wouldn't say dev-setup.md is deprecated. There are more details in the dev-setup.md that are useful if the the default setup in the makefile is not enough for testing. I think best would be to reference both in the testing documentation.

wallclockbuilder committed:
Update testing steps

Use both links for setting up. Make it clear one is more recent than the other.

wallclockbuilder committed:
Update link to Makefile

Makefile link is wrong as pointed out in the comments. Its correct location is one level higher than the wrong link.

wallclockbuilder commented:
Fixed.

ripcurlx commented on docs/testing.md:
```diff
@@ -1,13 +1,14 @@
 # Bisq Testing Guide
 
+
```
Could you please remove this empty line. Sometimes Travis gets stuck - in that case just leave a comment and I'll trigger a manual rebuild.

wallclockbuilder replied:
Done. Please rebuild.

wallclockbuilder replied:
continuous-integration/travis-ci/pr — The Travis CI build failed

ripcurlx replied:
Just re-triggered a Travis build

ripcurlx commented on docs/testing.md:
```diff
@@ -7,7 +7,7 @@ This guide describes the testing process performed prior to each release.
 
 - Build Bisq from source (see [build.md](build.md))
-- Setup a development/testing environment (see [dev-setup.md](dev-setup.md))
+- Setup a development/testing environment (see [Makefile](../Makefile) (new) or [dev-setup.md](dev-setup.md) (deprecated))
```
As mentioned already `dev-setup.md` is not deprecated. I would rather:
```suggestion
- Setup a development/testing environment (see [Makefile](../Makefile) or [dev-setup.md](dev-setup.md))
```

ripcurlx reviewed (changes requested):
NACK - Please see my comment.

wallclockbuilder committed:
Update docs/testing.md

Co-authored-by: Christoph Atteneder <christoph.atteneder@gmail.com>

ripcurlx reviewed (approved):
utACK
